### ISSUE TITLE
clone and run doesn't work

### ISSUE THREAD

**Issue by zivkan** [joelverhagen/NuGetTools#23]
The repo readme doesn't have instructions for getting started, so I tried cloning the repo, opened the sln and pressed F5. When my browser opens, I get this error message:

> ArgumentNullException: String reference not set to an instance of a String.
> Parameter name: s
> System.DateTimeParse.Parse(string s, DateTimeFormatInfo dtfi, DateTimeStyles styles, out TimeSpan offset)

Here's a screenshot of some more info:
![image](https://github.com/joelverhagen/NuGetTools/assets/5030577/0f0eecfe-f6e3-4d76-86ae-38a3627d752d)

**Comment by joelverhagen** [comment]
I think you need to run build.ps1 to initialize some stuff. Give that a try. I don't remember since it's been a long time since I've worked on this project any more than deploying new versions of NuGet client SDK.

**Comment by zivkan** [comment]
After a while I noticed the build.ps1. I still haven't run it, but I also see the `Build` app which seems to modify the assembly attributes of an already compiled dll. I haven't tried running build.ps1, but I'd assume that even if I did, building in VS will recreate the assembly without the attribute, so will fail again. So, I'm not sure how this app is supposed to be debuggable.

Anyway https://github.com/joelverhagen/NuGetTools/pull/22 looks like it fixes it.
